vms/platformvm/txs/executor/helpers_test.go

```diff
@@ -91,14 +77,35 @@ func (e *environment) SetState(blkID ids.ID, chainState state.Chain) {
 }
 
 func newEnvironment(t *testing.T, postBanff, postCortina, postDurango bool) *environment {
+	r := require.New(t)
+
 	var isBootstrapped utils.Atomic[bool]
 	isBootstrapped.Set(true)
 
-	config := defaultConfig(postBanff, postCortina, postDurango)
-	clk := defaultClock(postBanff || postCortina || postDurango)
+	var (
+		fork     ts.ActiveFork
+		forkTime time.Time
+	)
+	switch {
+	case postDurango:
+		fork = ts.DurangoFork
+		forkTime = ts.ValidateStartTime.Add(-2 * time.Second).Add(-2 * time.Second)
+	case postCortina:
+		fork = ts.CortinaFork
+		forkTime = ts.ValidateStartTime.Add(-2 * time.Second).Add(-2 * time.Second)
+	case postBanff:
+		fork = ts.BanffFork
+		forkTime = ts.ValidateEndTime.Add(-2 * time.Second)
+	default:
+		fork = ts.ApricotPhase5Fork
+		forkTime = ts.GenesisTime
+	}
```

Comment:
kind of arbitrary, but this is what we have been using fo far. Will be cleaned up in next PR https://github.com/ava-labs/avalanchego/pull/2169